README.md

```diff
@@ -9,46 +9,77 @@
-<a href="https://discord.gg/gBtUFKR">![Discord](https://img.shields.io/discord/490367152054992913.svg)</a>
+[![codecov][codecov-badge]][codecov-url]
+[![Discord chat][discord-badge]][discord-url]
+[![Telegram Group][telegram-badge]][telegram-url]
 
-* ⚖️ NEAR Protocol is a new smart-contract platform that delivers scalability and usability.
-* 🛠 Through sharding, it will linearly scale with the number of validation nodes on the network.
-* 🗝 Leveraging WebAssembly, TypeScript, more sane contract management, ephemeral accounts and many other advancements, NEAR
-finally makes using a blockchain protocol easy for both developers and consumers.
+master | beta | stable
+---|---|---|
+[![Build Status][ci-badge-master]][ci-url] | [![Build Status][ci-badge-beta]][ci-url] | [![Build Status][ci-badge-stable]][ci-url] 
 
-## Quick start
+[ci-badge-master]: https://badge.buildkite.com/a81147cb62c585cc434459eedd1d25e521453120ead9ee6c64.svg
+[ci-badge-beta]: https://badge.buildkite.com/a81147cb62c585cc434459eedd1d25e521453120ead9ee6c64.svg?branch=beta
+[ci-badge-stable]: https://badge.buildkite.com/a81147cb62c585cc434459eedd1d25e521453120ead9ee6c64.svg?branch=stable
+[ci-url]: https://buildkite.com/nearprotocol/nearcore
+[codecov-badge]: https://codecov.io/gh/nearprotocol/nearcore/branch/master/graph/badge.svg
+[codecov-url]: https://codecov.io/gh/nearprotocol/nearcore
+[discord-badge]: https://img.shields.io/discord/490367152054992913.svg
+[discord-url]: https://near.chat
+[telegram-badge]: https://cdn.jsdelivr.net/gh/Patrolavia/telegram-badge@8fe3382b3fd3a1c533ba270e608035a27e430c2e/chat.svg
+[telegram-url]: https://t.me/cryptonear
 
-[Check out our documentation](http://docs.nearprotocol.com), specifically:
- - [Working with Smart Contracts in NEAR Studio](https://docs.nearprotocol.com/docs/quick-start/development-overview)
+## About NEAR
 
-Develop and deploy contracts without any setup required using [NEAR Studio](https://studio.nearprotocol.com):
+NEAR's purpose is to enable community-driven innovation to benefit people around the world.
 
-[![NEAR Studio](https://github.com/nearprotocol/NEARStudio/blob/master/demos/guest_book.gif)](https://studio.nearprotocol.com)
+To achieve this purpose, *NEAR* provides a developer platform where developers and entrepreneurs can create apps that put users back in control of their data and assets, which is the foundation of ["Open Web" movement][open-web-url].
 
+One of the components of *NEAR* is NEAR Protocol, an infra for server-less applications and smart contracts powered by blockchain.
+NEAR Protocol is built to deliver usability and scalability of modern PaaS like Firebase at fraction of prices that blockchains like Ethereum charge.
 
-## Status
+*NEAR* overall provides wide range of tools for developers to easily build applications:
+ - [JS SDK][js-api] to connect ot NEAR Protocol from your applications.
```

Comment:
I would frame it as:

> Public JSON-RPC API with a first-party client library available for JavaScript and TypeScript.

Reply:
Hm, there is a bunch more stuff than just JSON RPC. Like Singers, KeyStores and Wallet support.

Reply:
Fair enough. Your last update sound more accurate :+1: